src/providers/VernaProvider.tsx

```diff
@@ -92,18 +100,20 @@ interface VernaContextProviderProps {
   objectFieldTemplate: ObjectFieldTemplateType;
   defaultSelector?: string;
   isEditor: boolean;
+  transformErrors?: (errors: AjvError[]) => AjvError[];
```

Comment:
I wonder if it would be safer to retrieve the type of transformErrors from `@rjsf` ?
In this way, if one day, @rjsf updates this method, we will be warned.

Reply:
Good idea !